CRM/Sqltasks/Action.php

```diff
@@ -185,7 +185,7 @@ public static function getAllActions($task) {
     // just compile list manually (for now)
 
     // add Segmentation Extension tasks (de.systopia.segmentation)
-    if (function_exists('segmentation_civicrm_install')) {
```

Comment:
This works with normal usage, but for test runs, where the segmentation extension might only be installed for one test and then uninstalled, `segmentation_civicrm_install` would still exist when the next test hits this code without the extension actually being installed. The new approach looks at the actual extension install status.